docs/install.sh

```diff
@@ -26,44 +25,67 @@ fi
 
 echo "INFO: verifying cpu architecture compatibility"
 
-ARCH=$(uname -m)
-ARCH=$(echo "${ARCH,,}") # convert to lower case
+ARCH=$(uname -m | tr '[:upper:]' '[:lower:]')
 
 if [ "$ARCH" = x86_64 ]; then
     ARCH="amd64"
 fi
+if [ "$ARCH" = aarch64 ]; then
+    ARCH="arm64"
+fi
 
-if [ "$ARCH" != "amd64" ]; then
+if [[ "$ARCH" != "amd64" && "$ARCH" != "arm64" ]]; then
```

Comment:
nit: inconsistent quotes around arch names `"arm64"` vs `aarch64` in lines 37 and 33 respectively